### ISSUE TITLE
Duplicate interface definition build errors when using Expo 44

### ISSUE THREAD

**Issue by chiubaka** [craftzdog/react-native-sqlite-2#117]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-sqlite-2@3.5.2` for the project I'm working on.

I was seeing build errors complaining about duplicate interface definitions similar to [this issue](https://github.com/expo/expo/issues/15622). I'm currently building with Expo 44.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-sqlite-2/ios/RNSqlite2.h b/node_modules/react-native-sqlite-2/ios/RNSqlite2.h
index bcaa915..3ba1d4c 100644
--- a/node_modules/react-native-sqlite-2/ios/RNSqlite2.h
+++ b/node_modules/react-native-sqlite-2/ios/RNSqlite2.h
@@ -1,10 +1,4 @@
-
-#if __has_include("RCTBridgeModule.h")
-#import "RCTBridgeModule.h"
-#else
 #import <React/RCTBridgeModule.h>
-#endif
-
 #import <Foundation/Foundation.h>
 
 #ifdef DEBUG_RNSqlite2
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by craftzdog** [comment]
Published as @3.6.0-beta.1. Can you please try it?

**Comment by chiubaka** [comment]
Apologies, I've chosen a different direction for my project in the past week and am no longer `react-native-sqlite-2`.
